src/DependencyInjection/Providers/KeycloakProviderConfigurator.php

```diff
@@ -57,6 +57,7 @@ public function getProviderOptions(array $config)
             'clientSecret' => $config['client_secret'],
             'authServerUrl' => $config['auth_server_url'],
             'realm' => $config['realm'],
+            'version' => $config['version'],
```

Comment:
It does seem to make sense - the `version` option was added in #382, but it was never exposed here.